src/console.rs

```diff
@@ -80,16 +80,16 @@ pub(crate) fn print<T: Into<Vec<u8>>>(msg: T) {
 }
 
 pub(crate) unsafe fn init(image_base: usize) -> anyhow::Result<()> {
-    let target_addr = transmute(image_base + 0x2e75f0);
+    let target_addr = transmute(image_base + 0x2fb970);//1.5.97: 0x2e75f0 -> 1.6.318: 0x2fba00 -> addressLib ID: 21890 -> 1.6.323: 0x2fb970
```

Comment:
Please remove those comments. I don’t think we need it

Reply:
I would have left them for reference to make it easier to find new addresses in casse Bethesda pushes another patch, but you decide.